solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -477,6 +484,11 @@ contract RandomBeacon is Ownable {
         }
     }
 
+    /// @notice Updates the minimum authorization amount.
+    function updateMinimumAuthorization(uint96 minAuthorization) external {
```

Comment:
`onlyOwner` is missing

Reply:
We should add a test for this function to `RandomBeacon.Parameters.test.ts`: revert + update.

Reply:
I'd also put them next to other update functions (`updateRelayEntryParameters`, `updateGroupCreationParameters`, `updateDkgParameters`, `updateRewardParameters`). Maybe right after ctor? It feels like a major `update*` function.

Reply:
ff369a8